src/app/pages/help/index.js

```diff
@@ -130,7 +148,13 @@ const Help = () => {
 				<div className="resources-search">
 					<Search />
 					<div className="button-container">
-						<Button href="https://www.bluehost.com/resources/" isLink isSecondary>
+						<Button
+							href="https://www.bluehost.com/resources/"
+							isLink
```

Comment:
The doubling up of `isLink` and `isSecondary` gets resolved in #61, but may need a rebase after this linting.

Reply:
Updated